pkg/config/vars.go

```diff
@@ -277,6 +277,7 @@ func setKwokctlKwokConfig(conf *v1alpha1.KwokctlConfigurationOptions) {
 		conf.KwokControllerImage = joinImageURI(conf.KwokImagePrefix, "kwok", conf.KwokVersion)
 	}
 	conf.KwokControllerImage = envs.GetEnvWithPrefix("CONTROLLER_IMAGE", conf.KwokControllerImage)
+	conf.KwokControllerPort = envs.GetEnvWithPrefix("CONTROLLER_PORT", conf.KwokControllerPort)
```

Comment:
Is it okay to extend configuration with this env variable? I think I followed conventions :relaxed:

Reply:
This should be done in a new PR and needs to be considered for each runtime to expose this port correctly on the host, see PrometheusPort.